Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift

```diff
@@ -0,0 +1,366 @@
+            product: Products.blackCoralShades,
+            customer: Customers.PaytinLubin,
+            status: .hold,
+            text: "Travel in style!",
+            rating: 5,
+            verified: true
+        ),
+        createProductReview(
+            product: Products.blackCoralShades,
+            customer: Customers.LydiaDonin,
+            status: .hold,
+            text: "I love these shades! They are super sturdy and stylish at the same time. I'm planning to buy more of them!",
+            rating: 5,
+            verified: false
+        ),
+        createProductReview(
+            product: Products.blackCoralShades,
+            customer: Customers.MiraWorkman,
+            status: .hold,
+            text: "Can't go outside without them anymore!",
+            rating: 5,
+            verified: false
+        ),
+        createProductReview(
+            product: Products.akoyaPearlShades,
+            customer: Customers.MarcusCurtis,
+            status: .approved,
+            text: "Such a great product!",
+            rating: 4,
+            verified: true
+        ),
+        createProductReview(
+            product: Products.coloradoShades,
+            customer: Customers.LucyLiu,
+            status: .approved,
+            text: "Great UV protection, I've dropped them a few times and they are still holding up perfectly",
+            rating: 5,
+            verified: true
+        ),
+        createProductReview(
+            product: Products.roseGoldShades,
+            customer: Customers.JaneCheng,
+            status: .approved,
+            text: "I love these shades! They are super sturdy and stylish at the same time. I’m planning to buy them again for a birthday gift next month.",
+            rating: 5,
+            verified: true
+        ),
+        createProductReview(
+            product: Products.roseGoldShades,
+            customer: Customers.ChanceVicarro,
+            status: .approved,
+            text: "The best I've ever worn",
+            rating: 4,
+            verified: true
+        )
+    ]
+
+    public let thisYearVisitStats: SiteVisitStats = createVisitStats(
+        granularity: .year,
+        items: [
+            createVisitStatsItem(
+                granularity: .year,
+                ago: 0,
+                visitors: Int.random(in: 100 ... 1000)
+            )
+        ]
+    )
+
+    public var thisYearTopProducts: TopEarnerStats = createStats(granularity: .year, items: [
+        createTopEarningItem(product: Products.akoyaPearlShades, quantity: 17),
+        createTopEarningItem(product: Products.blackCoralShades, quantity: 11),
+        createTopEarningItem(product: Products.coloradoShades, quantity: 5),
+    ])
+
+    /// The probability of a sale for each visit when generating random stats
+    ///
+    let orderProbabilityRange = 0.1...0.5
+
+    /// The possible value of an order when generating random stats
+    ///
+    let orderValueRange = 100 ..< 500
```

Comment:
ℹ️  These can be `private`. 🙂 

```suggestion
    /// The probability of a sale for each visit when generating random stats
    ///
    private let orderProbabilityRange = 0.1...0.5

    /// The possible value of an order when generating random stats
    ///
    private let orderValueRange = 100 ..< 500
```

Reply:
Sure can – also addressed in c99bc531f92a9b1b4fdbb3ba6e5b9eb5efc108b0